client/ionic/src/App.tsx

```diff
@@ -37,30 +34,15 @@ import './theme/variables.css';
 const App: React.FC = () => (
   <IonApp>
     <IonReactRouter>
-      <IonTabs>
         <IonRouterOutlet>
-          <Route path="/tab1" component={Tab1} exact={true} />
-          <Route path="/tab2" component={Tab2} exact={true} />
+          <Route path="/about" component={About} exact={true} />
+          <Route path="/menu" component={Menu} exact={true} />
           <Route path="/triage" component={Triage} />
-          <Route path="/" render={() => <Redirect to="/tab1" />} exact={true} />
+          <Route path="/" render={() => <Redirect to="/about" />} exact={true} />
```

Comment:
not super important: but it might be nicer to add the `<Redirect>` tag as the child to this `Route` component, as in this example: https://reacttraining.com/react-router/web/api/Redirect

Reply:
can you add as an issue?

Reply:
fixed with rebase